src/dotnet/Chat.UI.Blazor/Components/ChatView/Items/NavigateToChatEntryEvent.cs

```diff
@@ -0,0 +1,3 @@
+namespace ActualChat.Chat.UI.Blazor.Components;
+
+public record NavigateToChatEntryEvent(long ChatEntryId);
```

Comment:
It's named like an event but seems like a command
https://jimmybogard.com/message-naming-conventions/#:~:text=Event%20naming%20conventions,Subject

Reply:
removed suffix